### ISSUE TITLE
Lint/PercentStringArray or Style/PercentLiteralDelimiters breaks my code

### ISSUE THREAD

**Issue by Dorian** [rubocop-hq/rubocop#5833]
--------

## Expected behavior

Doesn't break my code.

Makes it into:

```
config.csp = {
  default_src: %w['self']
}
```

## Actual behavior

Breaks my code.

Makes it into:

```
config.csp = {
  default_src: %w[self]
}
```

## Steps to reproduce the problem

```
config.csp = {
  default_src: %W('self')
}
```

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.55.0 (using Parser 2.5.1.0, running on ruby 2.4.1 x86_64-darwin17)
```

**Comment by koic** [comment]
Yes. As @composerinteralia commented on, this is the intended behavior. I close this issue, but please feedback if there is a good solution. Thank you!

**Comment by mediafinger** [comment]
The intended behavior is misleading. I also did expect `default` or `EnforcedStyle` to influence all literal delimeters. I feels like a bug from my point of view, as I had to add all of them to the config file:

```yaml
Style/PercentLiteralDelimiters:
  PreferredDelimiters:
    default:  ()
    "%":  ()
    "%i": ()
    "%q": ()
    "%Q": ()
    "%r": "{}"
    "%s": ()
    "%w": ()
    "%W": ()
    "%x": ()
```
